charts/immich/Chart.yaml

```
@@ -2,8 +2,8 @@
 apiVersion: v2
 description: A chart to power Immich (immich.app) running on kubernetes
 name: immich
-version: 0.4.0
-appVersion: v1.95.1
+version: 0.4.1
```

I think this warrants a minor version bump.

```suggestion
version: 0.5.0
```